.travis.yml

```diff
@@ -191,7 +191,12 @@ script:
       Xvfb :1 -screen 0 1024x768x24 &
       xfwm4 &
       export ASAN_OPTIONS=detect_leaks=0;
-      ./test-runner
+      if ! ./test-runner ; then
+        if [ -f "/tmp/enigma_draw_diff.png" ]; then
+          ./CommandLine/testing/GitHub-ImageDiff.sh
```

Comment:
I am minorly disappointed that this is a single file; I was hoping the harness diff'd all output. However, this is still a great start. Let's run with it for now.